packages/theme/pages/Checkout/Shipping.vue

```diff
@@ -22,6 +26,8 @@
           slim
         >
           <SfInput
+            v-on:click="getBackToShippingDetails()"
+            v-bind:class="{'disbale-input': isFormSubmitted}"
```

Comment:
This seems like a typo

```suggestion
            :class="{'disable-input': isFormSubmitted}"
```